assignments/callbacks.js

```diff
@@ -27,23 +27,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(first) {
+  console.log(first)
+});
+
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3]);
```

Comment:
If you passed 3 in and had an array with 10 elements in it, that wouldn't pass the last item of the array into the callback.

To get the last element in, you'd want to use `arr[arr.length - 1]`.